docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb

```diff
@@ -0,0 +1,206 @@
+      "Activating auto-logging. Current session state plus future input saved.\n",
+      "Filename       : C:\\Users\\v-singak\\.qcodes\\logs\\command_history.log\n",
+      "Mode           : append\n",
+      "Output logging : True\n",
+      "Raw input log  : False\n",
+      "Timestamping   : True\n",
+      "State          : active\n",
+      "Qcodes Logfile : C:\\Users\\v-singak\\.qcodes\\logs\\201203-3532-qcodes.log\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import numpy as np\n",
+    "\n",
+    "import qcodes as qc\n",
+    "from qcodes import Station, initialise_or_create_database_at, \\\n",
+    "    load_or_create_experiment, Measurement\n",
+    "from qcodes.tests.instrument_mocks import DummyInstrument, \\\n",
+    "    DummyInstrumentWithMeasurement\n",
+    "\n",
+    "qc.logger.start_all_logging()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this notebook, we create a mock station setup. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-12-03T16:58:16.814459Z",
+     "start_time": "2020-12-03T16:58:16.802462Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "dac = DummyInstrument('dac', gates=['ch1', 'ch2'])\n",
+    "dmm = DummyInstrumentWithMeasurement(name='dmm', setter_instr=dac)\n",
+    "\n",
+    "station = qc.Station(dmm, dac)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After that, database needs to be initialized or created if it doesn't exist using `initialize_or_create_database` method. Furthermore, datasets are associated with experiments and by default the run is appended to the latest existing experiment. We can load or create an experiment using `load_or_create_experiment` function. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-12-03T17:20:12.074558Z",
+     "start_time": "2020-12-03T17:20:12.046560Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "db_file_path = os.path.join(os.getcwd(), 'plottr_for_live_plotting_tutorial.db')\n",
+    "initialise_or_create_database_at(db_file_path)\n",
+    "exp = load_or_create_experiment(experiment_name='plottr_for_live_plotting_with_subsecond_refresh_rate',\n",
+    "                          sample_name=\"no sample\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now open an anaconda prompt and activate your conda environment where plottr is installed. Start Inspectr GUI with `plottr-inspectr` command. You will see a window as show below. In this window, set `Refresh interval (s)` to the desired value and check `Auto-plot new`. Using `File` button load the database initialized (or created) above.\n",
+    "\n",
+    "<div>\n",
+    "<img src=\"files/plottr.jpg\" width=\"400\" height=\"400\"/>\n",
```

Comment:
you could choose to store the static files one level up, at the `docs/examples/files` directiory, for consistency (?)